openwisp_controller/config/apps.py

```diff
@@ -320,6 +321,56 @@ def register_dashboard_charts(self):
                 'labels': {'': _('undefined')},
             },
         )
+        if app_settings.GROUP_PIE_CHART:
+            register_dashboard_chart(
+                position=13,
+                config={
+                    'name': _('Groups'),
+                    'query_params': {
+                        'app_label': 'config',
+                        'model': 'devicegroup',
+                        'annotate': {
+                            'with_device': Count(
```

Comment:
can you please chagne `with_device` to `active`?
Applies to similar occurrences of this naming.